dom.bs

```diff
@@ -2382,17 +2412,11 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
    <li><p>Otherwise, <a for=set>insert</a> <var>node</var> into <var>parent</var>'s
    <a for=tree>children</a> before <var>child</var>'s <a for=tree>index</a>.
 
-   <li><p>If <var>parent</var> is a <a for=Element>shadow host</a> and <var>node</var> is a
-   <a>slotable</a>, then <a>assign a slot</a> for <var>node</var>.
-
    <li>If <var>node</var> is a {{Text}} node, run the <a>child text content change steps</a> for
    <var>parent</var>.
 
-   <li><p>If <var>parent</var>'s <a for=tree>root</a> is a <a for=/>shadow root</a>, and
-   <var>parent</var> is a <a>slot</a> whose <a for=slot>assigned nodes</a> is the empty list,
-   then run <a>signal a slot change</a> for <var>parent</var>.
-
-   <li><p>Run <a>assign slotables for a tree</a> with <var>node</var>'s <a for=tree>root</a>.
+   <li><p>If the <i>suppress observers flag</i> is unset, then <a>update slots</a> with « »,
+   <var>parent</var>, and « <var>node</var> ».
```

Comment:
Why do we need to run it on both `parent` and `node`? We just inserted `node` into the same tree as `parent`.

Reply:
That's not what this does, right? It uses parent for different things then node.